frontend/src/metabase/visualizations/lib/settings/graph.js

```diff
@@ -318,8 +305,7 @@ export const GRAPH_DISPLAY_VALUES_SETTINGS = {
     section: t`Display`,
     title: t`Show values on data points`,
     widget: "toggle",
-    getHidden: (series, vizSettings) =>
-      vizSettings["stackable.stack_type"] === "normalized",
+    getHidden: canHaveDataLabels,
```

Comment:
```suggestion
    getHidden: !canHaveDataLabels,
```

I believe the check should be `!canHaveDataLabels`, right? We want to hide the setting only if they cannot have data labels?